source/reference.txt

```diff
@@ -75,10 +75,13 @@ Reference
 :doc:`/reference/server-sessions`
    Describes server sessions.
 
+:doc:`/reference/versioned-api`
+   Describes the Versioned API.
 
-.. seealso::
+.. seealso:: 
 
-   The
+   The :ref:`genindex` may provide useful insight into the
```

Comment:
I don't see this ref in this PR and I can't find it anywhere in the v5.0 branch. Should this be here?